### PR TITLE
minor updates

### PR DESCRIPTION
**Pull request by stormcloud266**
Knocks out a few smol changes that were requested:

* Decrease this space below the Send logo and menu. Cut it in half
* remove notifications, security, privacy from settings
* Fix alignment on subtext rewards page

### CODE REVIEW

**Comment by stormcloud266** [comment]
* **#693** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/693?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `dev`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @stormcloud266 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/app/features/account/sendtag/checkout/checkout-form.tsx [hunk 1]
```diff
@@ -378,7 +378,10 @@ function ReferredBy() {
             <Input
               id={'refcode'}
               defaultValue={referralCode ?? ''}
-              onChangeText={(text) => mutation.mutate(text)}
+              onChangeText={(text) => {
+                console.log('text: ', text)
```
Comment (youngkidwarrior) [inline packages/app/features/account/sendtag/checkout/checkout-form.tsx:382]: Oops

File: packages/app/features/account/settings/SettingsLinks.tsx [hunk 2]
```diff
@@ -12,20 +12,20 @@ export function SettingsLinks(): JSX.Element {
...
-    {
-      text: 'Security',
-      href: '/account/settings/security',
-      disabled: true,
-    },
-    {
-      text: 'Privacy',
-      href: 'https://info.send.it/legal/privacy-policy',
-    },
+    // {
```
Comment (youngkidwarrior) [inline packages/app/features/account/settings/SettingsLinks.tsx:15]: Just go ahead and remove them. We can add them back later if we want

**Review by youngkidwarrior — APPROVED** [review]
Couple lil things

File: packages/app/features/account/sendtag/checkout/checkout-form.tsx (hunk 1 above)
Comment (stormcloud266) [reply packages/app/features/account/sendtag/checkout/checkout-form.tsx:382]: omg

**Comment by youngkidwarrior** [comment]
### Merge activity

* **Sep 3, 6:38 PM EDT**: @youngkidwarrior started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/0xsend/sendapp/693).

**Pull request merged**